src/js/components/PageHeader/stories/Size.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import {
+  Anchor,
+  Form,
+  FormField,
+  PageHeader,
+  Page,
+  PageContent,
+  Select,
+} from 'grommet';
+
+export const Size = () => {
+  const [size, setSize] = useState('medium');
```

Comment:
Wondering if we should always show all sizes, similar to how Heading:All and Paragraph:All work. I suggest this partly because Chromatic would detect any changes if we did that, which it wouldn't detect when requiring interaction.

Reply:
Done.